CodeEdit/Settings/GeneralSettingsView.swift

```diff
@@ -15,6 +15,7 @@ struct GeneralSettingsView: View {
     @AppStorage(ReopenBehavior.storageKey) var reopenBehavior: ReopenBehavior = .default
     @AppStorage(FileIconStyle.storageKey) var fileIconStyle: FileIconStyle = .default
     @AppStorage(CodeFileView.Theme.storageKey) var editorTheme: CodeFileView.Theme = .atelierSavannaAuto
+    @AppStorage("defaultTabWidth") var defaultTabWidth: Int = 4
```

Comment:
Please don't use String literals as identifiers for `UserDefaults`. In this case just use a case-less `enum` containing the default value and a `storageKey`.

Example: 
```swift
public enum EditorTabWidth {
	static public let `default`: Int = 4 
	static public let storageKey = "defaultTabWidth"
}
```

Reply:
Good suggestion! Added in cd76e3460356333e35bfb4d48f76025019307198.